docs/data-sources/group.md

```diff
@@ -38,6 +38,7 @@ The following arguments are supported:
 
 The following attributes are exported:
 
+* `administrative_unit_id` - The optional administrative unit ID that the group is member of.
```

Comment:
I don't think we should include this in the data source, since a group could be a member of any number of AUs including none.
```suggestion
```